### PR TITLE
2021q2 foundation report

### PR DESCRIPTION
**Pull request by emaste**

### CODE REVIEW

File: 2021q2/freebsd-foundation.adoc [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+First, we’d like to say thank you to everyone who has given us a financial contribution this year!
+Last quarter we raised $70,410, which includes donations from organizations like Verisign, VMware, and Stormshield, as well as many individuals.
+
+We depend on these donations to fund our work supporting FreeBSD.
+Late last year we decided to put more funding into helping to improve FreeBSD.
+We hired a Sr. Software Developer to work on arm64 and a Project Coordinator to manage our projects and interface with the community.
+We also hired two of our part-time software developers full-time.
+The purpose of this was to provide more resources to step in to implement and improve major features in FreeBSD, review patches and bug reports, implement bug fixes, and support the security efforts.
+This keeps FreeBSD the innovative, secure, and reliable operating system you rely on.
```
Comment (pauamma) [inline 2021q2/freebsd-foundation.adoc:35]: "reliable operating system you rely on" reads a bit awkward to me. Perhaps "... you use"?

File: 2021q2/freebsd-foundation.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Find out more information and share with your companies! https://www.freebsdfoundation.org/FreeBSD-foundation-partnership-program/
+
+==== OS Improvements
+
+==== Continuous Integration and Quality Assurance
+
+==== Supporting FreeBSD Infrastructure
+
+The Foundation provides hardware and support to improve the FreeBSD infrastructure.
+Last quarter, we supported the text cluster at Sentex, purchased a few needed parts for infrastructure in general, and started working with the clusteradm team on a more efficient and improved hardware request/purchase process.
```
Comment (pauamma) [inline 2021q2/freebsd-foundation.adoc:91]: "text" = "test"? (If not, please explain what it is.)

File: 2021q2/freebsd-foundation.adoc [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+This includes promoting work being done by others with FreeBSD; producing advocacy literature to teach people about FreeBSD and help make the path to starting using FreeBSD or contributing to the Project easier; and attending and getting other FreeBSD contributors to volunteer to run FreeBSD events, staff FreeBSD tables, and give FreeBSD presentations.
+
+The FreeBSD Foundation sponsors many conferences, events, and summits around the globe.
+These events can be BSD-related, open source, or technology events geared towards underrepresented groups.
+We support the FreeBSD-focused events to help provide a venue for sharing knowledge, to work together on projects, and to facilitate collaboration between developers and commercial users.
+This all helps provide a healthy ecosystem.
+We support the non-FreeBSD events to promote and raise awareness of FreeBSD, to increase the use of FreeBSD in different applications, and to recruit more contributors to the Project.
+While we were still unable to attend in-person meetings due to covid-19, we were able to attend virtual events and help organize the June 2021 FreeBSD Developer Summit.
```
Comment (pauamma) [inline 2021q2/freebsd-foundation.adoc:103]: Maybe "Covid"? I think most uses I've seen in English had it titlecapped.

File: 2021q2/freebsd-foundation.adoc (hunk 1 above)
Comment (emaste) [reply 2021q2/freebsd-foundation.adoc:35]: Will take a stab at an edit

**Comment by emaste** [comment]
Anne reports some of the links in the advocacy section disappeared, looking...

**Comment by emaste** [comment]
Links from @adickison in https://github.com/emaste/freebsd-quarterly/pull/10 added back. (The pull request reverted the changes I made on @pauamma's suggestion so I just copied the new links over).

**Pull request closed without merging**